data/startup.sh

```diff
@@ -47,7 +47,7 @@ exit_workdir() {
 setup() {
   echo "Setting up the environment for preloading..."
   if systemctl status update-engine; then
-    systemctl stop update-engine
+    systemctl stop update-engine || :
```

Comment:
A "stop_service" function was added recently; you might be able to use that here.